### PR TITLE
Fix/css

### PR DESCRIPTION
**Pull request by ghost**
[X] - Fix
[X] - Refactor

Added SCSS variables and mixin; add responsive styling; add tabindexes to allow user to tab through site

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -9,46 +9,45 @@
...
-        <p id="userStridelength"></p>
-        <p id="stepGoalCard"></p>
-        <p id="avStepGoalCard"><p>
-        <p>Friends</p>
-        <ul class="card-list" id="friendList"></ul>
-        <p>Keep up the good work! You were increasingly active on these dates:</p>
-        <ul class="card-list" id="streakListMinutes"></ul>
+      <section class="sidebar" tabindex="1">
+        <h2 id="sidebarName" tabindex="2"></h2>
+        <img src="./images/the-rock.jpg" alt="Dwayne 'The Rock' Johnson, smiling at the camera">
```
Comment (shayan-golafshani) [inline dist/index.html:14]: Nice job cleaning up unnecessary classes.

File: src/css/base.scss [hunk 2]
```diff
@@ -1,6 +1,15 @@
-/* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: pink;
 
-body {
-  background-color: $primary-background;
+$primary-background: #4d4e4f;
```
Comment (shayan-golafshani) [inline src/css/base.scss:2]: Sass variables are very intuitive

File: src/css/base.scss [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+$sidebar-background: #9A9DAB;
+$card-background: #FFFFFF;
+$hydration-card: #5BC8AC;
+$activity-card: #DFE166;
+$sleep-card: #EC96A4;
+$border: 1px solid #000000;
+$primary-font: 'Open Sans', sans-serif;
+$secondary-font: sans-serif;
+
+@mixin flexed-element($direction) {
```
Comment (shayan-golafshani) [inline src/css/base.scss:12]: Nice mixin for flex-box! Could add justify-content as an argument to this mixin.

**Comment by BigBike96** [comment]
I like the names you used for the variables

File: src/css/styles.scss [hunk 4]
```diff
@@ -1,31 +1,39 @@
...
+h3 {
+  font-size: 1em;
+}
+
 main {
-  display: flex;
-  flex-direction: row;
-  height: 2000px;
+  @include flexed-element(row);
+  height: 100%;
```
Comment (shayan-golafshani) [inline src/css/styles.scss:28]: Nice n responsive.

File: src/css/styles.scss [hunk 5]
```diff
@@ -34,91 +42,109 @@ main {
...
-  border: 1px solid black;
-  display: flex;
+  background-color: $card-background;
+  border: $border;
   margin: 1%;
 }
 
 .horizontal-card {
-  flex-direction: column;
+  @include flexed-element(column);
```
Comment (shayan-golafshani) [inline src/css/styles.scss:80]: Could add justify content as another argument to the mixin

File: src/css/styles.scss [hunk 6]
```diff
@@ -34,91 +42,109 @@ main {
...
-  font-family: sans-serif;
-  display: flex;
-  flex-direction: column;
+  font-family: $secondary-font;
+  @include flexed-element(column);
   text-align: center;
   padding: 2%;
   width: 80%;
+
+  &:hover {
```
Comment (shayan-golafshani) [inline src/css/styles.scss:101]: Nice use of nesting

File: src/css/base.scss (hunk 3 above)
Comment (BigBike96) [reply src/css/base.scss:12]: Mixin could also be used for media queries

File: src/css/styles.scss [hunk 7]
```diff
@@ -34,91 +42,109 @@ main {
...
   list-style-type: none;
   font-weight: normal;
-  font-family: sans-serif;
+  font-family: $secondary-font;
   font-style: italic;
   padding: 1em;
   font-size: 1em;
 }
 
+@media only screen and (max-width: 1330px) {
```
Comment (shayan-golafshani) [inline src/css/styles.scss:135]: NIce media queries

**Review by shayan-golafshani — APPROVED** [review]
Good job

**Pull request merged**